r4/constants.bal

```diff
@@ -34,3 +34,5 @@ public const FHIR_BASE_IG = "international";
 
 # This is the regex for validating canonical URLs
 const CANONICAL_REGEX = "^(https?|http)\\:\\/\\/[-a-zA-Z0-9+&@#\\/%~_!:,\\.;]*[-a-zA-Z0-9\\+&@#\\/%=~_](\\|[-a-zA-Z0-9]*)?";
+
+const REFERENCE_TYPE_CONTAINED = "contained";
```

Comment:
```suggestion
const REFERENCE_TYPE_CONTAINED = "contained";

```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/3888607aaff6cbb7160216d517dab62dbc505132